doc/INSTALL_AND_RUN.md

```diff
@@ -140,13 +140,13 @@ git clone https://github.com/SimplyVC/polkadot_api_server
 Then run the following commands to build the image:
 ```bash
 cd polkadot_api_server
-docker build -t simplyvc/polkadot_api_server:1.13.1 .
+docker build -t simplyvc/polkadot_api_server:1.16.1 .
```

Comment:
Can we consider using latest? I assume users are always interested in using the latest one.

Reply:
Because the `:latest` flag can be ambiguous.
This is explained in [this article](https://blog.container-solutions.com/docker-latest-confusion)